### PR TITLE
Add Ubuntu SSL Salt-Files

### PR DESCRIPTION
**Pull request by amannch**
Installs SSL-Certificate on Ubuntu minions at bootstrap.
Works for 18.04 but sets a fallback for Debian systems
(Fallback tested for Debian 9: works!)

## What does this PR change?

Adds Ubuntu bootstrap SLS files

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed

- [X] **DONE**

## Test coverage
- No tests: Works on my sumaform instance

- [X] **DONE**

## Links

https://github.com/SUSE/spacewalk/issues/6364

- [X] **DONE**

### CODE REVIEW

File: susemanager-utils/susemanager-sls/salt/certs/init.sls [hunk 1]
```diff
@@ -19,4 +19,10 @@
 {% else -%}
 {{ includesls(grains['os'], grains['osrelease_info']|first|string) }}
 {% endif -%}
+{% elif grains['os_family'] == 'Debian' %}
+{% if grains['os'] == 'Ubuntu' %}
```
Comment (mateiw) [inline susemanager-utils/susemanager-sls/salt/certs/init.sls]: Maybe this condition can be removed. I think this is generic enough to work on both stock Debian and Ubuntu.

Comment (amannch) [reply susemanager-utils/susemanager-sls/salt/certs/init.sls]: makes sense, it's now the same for Debian and Ubuntu

**Comment by mateiw** [comment]
gitarro no changelog needed !!!

**Comment by amannch** [comment]
gitarro rerun changelog_test !!!

**Pull request merged**